src/test/java/org/takes/rq/CapInputStreamTest.java

```diff
@@ -46,13 +46,13 @@
      */
     @Test
     public void putsCapOnStream() throws IOException {
-        final long length = 50L;
+        final byte[] bytes = "test1".getBytes();
         MatcherAssert.assertThat(
             (long) new CapInputStream(
```

Comment:
@prondzyn  this will return an `int` which is enough as `bytes.length` is an `int` such that you can avoid casting it as a `long`